core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -1220,9 +1220,12 @@ public void testAsyncPgExecutesTableDoesNotExists() throws Exception {
                 ">700000000a717565737400\n" +
                 "<520000000800000000530000001154696d655a6f6e6500474d5400530000001d6170706c69636174696f6e5f6e616d6500517565737444420053000000187365727665725f76657273696f6e0031312e33005300000019696e74656765725f6461746574696d6573006f6e005300000019636c69656e745f656e636f64696e670055544638004b0000000c0000003fbb8b96505a0000000549\n" +
                 ">500000003e5f5f6173796e6370675f73746d745f315f5f0053454c454354202a2046524f4d20746869737461626c65646f65736e6f7465786973743b0000004400000018535f5f6173796e6370675f73746d745f315f5f004800000004\n" +
-                "<450000004b433030303030004d7461626c6520646f6573206e6f74206578697374205b7461626c653d746869737461626c65646f65736e6f7465786973745d00534552524f520050313500005a0000000549\n" +
-                ">5300000004510000004753454c4543542070675f61647669736f72795f756e6c6f636b5f616c6c28293b0a434c4f534520414c4c3b0a554e4c495354454e202a3b0a524553455420414c4c3b00\n" +
-                "<540000002f000170675f61647669736f72795f756e6c6f636b5f616c6c0000000000000100000413ffffffffffff0000440000000a0001ffffffff430000000d53454c4543542031004300000008534554004300000008534554004300000008534554005a0000000549\n";
+                "<450000004b433030303030004d7461626c6520646f6573206e6f74206578697374205b7461626c653d746869737461626c65646f65736e6f7465786973745d00534552524f52005031350000\n" +
+                ">5300000004\n" +
+                "<5a0000000549\n" +
```

Comment:
The old behaviour: On error, QuestDB would send an error message followed immediately by ReadyForQuery(IDLE)
The new behaviour: On error, QuestDB sends an error message only. The server sends ReadForQuery(IDLE) is only upon receiving a SYNC message from a client. That's what the pgwire protocol mandates. 

This is meant to be an explanation for reviewers, feel free to resolve it upon reading.